sensing/pointcloud_preprocessor/src/blockage_diag/blockage_diag_nodelet.cpp

```diff
@@ -150,24 +157,16 @@ void BlockageDiagComponent::filter(
   std::scoped_lock lock(mutex_);
   int vertical_bins = vertical_bins_;
   int ideal_horizontal_bins;
-  float distance_coefficient = 327.67f;
-  float horizontal_resolution_ = 0.4f;
-  if (lidar_model_ == "Pandar40P") {
-    distance_coefficient = 327.67f;
-    horizontal_resolution_ = 0.4f;
-  } else if (lidar_model_ == "PandarQT") {
-    distance_coefficient = 3276.75f;
-    horizontal_resolution_ = 0.6f;
+  double compensate_angle = 0.0;
+  if (angle_range_deg_[0] > angle_range_deg_[1]) {
```

Comment:
[Q] Does angle_range_deg_[0]  mean minimum_angle? and angle_range_deg_[1] means maximum_angle?
The code is a bit hard to understand. Could you please add comments or change variable names to more descriptive ones?